lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -1075,6 +1075,7 @@ int ReadDxcOpts(const OptTable *optionTable, unsigned flagsToInclude,
         opts.SpirvOptions.debugInfoLine = true;
         opts.SpirvOptions.debugInfoRich = true;
       } else if (v == "vulkan") {
+		// For test purposes only
```

Comment:
nit: alignment of comment, period at end of comment

Reply:
Gah! Fixed.